File: src/test/java/aoc2024/Day03Test.java
package aoc2024;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Day03Test {

    @Test
    void aoc_example_part1() {
        assertEquals("161", new Day03().asTest().withInputFileSuffix("_1").solvePart1());
    }

    @Test
    void aoc_example_part2() {
        assertEquals("48", new Day03().asTest().withInputFileSuffix("_2").solvePart2());
    }
}

File: src/test/java/aoc2025/Day03Test.kt
package aoc2025

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class Day03Test {

    @Test
    fun aoc_example_part1() {
        assertEquals("357", Day03().asTest().solvePart1())
    }

    @Test
    fun aoc_example_part2() {
        assertEquals("3121910778619", Day03().asTest().solvePart2())
    }
}
